Find a functionally equivalent code:android.widget.DatePicker.getCalendarViewShown:COMMENT
Method Modifier: public      
Comment:/**
 * Gets whether the {@link CalendarView} is shown.
 *
 * @return True if the calendar view is shown.
 * @see #getCalendarView()
 */

@@ -1,3 +1,3 @@
 {
-    return (mCalendarView.getVisibility() == View.VISIBLE);
+    return mDelegate.getCalendarViewShown();
 }

